javascript/builtins/Intl.json

```diff
@@ -869,7 +869,7 @@
                 "version_added": false
               },
               "samsunginternet_android": {
-                "version_added": true
+                "version_added": false
```

Comment:
I don't trust the prepopulated data or the autogenerated data...

Can we test this?